src/cast_core/src/QuiesceActor.cpp

```diff
@@ -65,7 +65,16 @@ QuiesceActor::QuiesceActor(genny::ActorContext& context)
     : Actor{context},
       _totalQuiesces{context.operation("Quiesce", QuiesceActor::id())},
       _client{context.client()},
-      _loop{context} {}
+      _loop{context} {
+
+    std::string srv = "mongodb+srv://";
+    auto uri_prefix = _client->uri().to_string().substr(0, srv.length());
+
+    if (uri_prefix == srv) {
```

Comment:
(Nit: `urlPrefix` since genny uses camelCase)
Also nit: I think you can use `str::find` to look for the srv thing. Might be cleaner.

```
if (_client->uri().to_string().find("...srv") == 1) // found at beginning
```

(At least I think that's right.)

Just a nit. Otherwise looks fine.

Reply:
Will change to camelCase.

I chose (`sub_str` + `==`) instead of `find` because I had a theory that the later might be faster :) . But here the performance difference between these two approach are negligible as the strings are short and is not a concern at all because this is done during initialisation. 

Therefore, after a second thought, I agree with your that using `find` to make it more intuitive is better.